helm/ocs-monkey-chaos/values.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# Container image for the workload controller
+controllerImage: quay.io/dbindra/ocs-monkey-chaos-runner:may12_2
```

Comment:
This should point to the same container image used by the workload generator since you're using the same container image for both.